app-frontend/src/components/AuthPage/__tests__/LogIn.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import LogIn, { LogInProps } from '../LogIn';
+
+function noop() {}
+
+const props: LogInProps = {
+  loading: false,
+  error: '',
+  authorizationHandler: jest.fn(noop),
+  changeHandler: jest.fn(noop),
+};
+
+const setUp = () => shallow(
+  <LogIn
+    loading={props.loading}
+    error={props.error}
+    authorizationHandler={props.authorizationHandler}
+    changeHandler={props.changeHandler}
+  />,
+);
+
+describe('<LogIn />', () => {
+  it('Should call an authorization function after pressing Log in', () => {
+    const component = setUp();
+    const logInButton = component.find('.authorize-button');
+
+    logInButton.simulate('click');
+    expect(props.authorizationHandler).toHaveBeenCalledTimes(1);
+  });
+
+  it('Should update inputs when user types something', () => {
+    const component = setUp();
+    const logInEmailInput = component.find('#user-email');
```

Comment:
if you added ids by purpose, then it is not necessary.
Here are all ways how you can select the desired component
https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/find.html

Reply:
this was added in purpose of markup, not for using in tests.